pom.xml

```diff
@@ -439,7 +439,7 @@
     <jetty.version>9.4.24.v20191120</jetty.version>
     <jackson.version>2.10.3</jackson.version>
     <json.version>1.1.1</json.version>
-    <jline.version>2.14.6</jline.version>
+    <jline.version>3.6.2</jline.version>
```

Comment:
Why choose the `3.6.2(Mar, 2018)`, instead [3.16.0(Jul, 2020)](https://mvnrepository.com/artifact/org.jline/jline/3.16.0)?

Reply:
Thank you @maoling for the code review. I use this 3.6.2 version while my product use this. Of course, you are right. I have push a new commit to upgrade jline version to `3.16.0`, BTW, I have test this new version in my production environment.